app/jobs/mailchimp_nonprofit_user_job.rb

```diff
@@ -0,0 +1,7 @@
+class MailchimpNonprofitUserJob < ActiveJob::Base
```

Comment:
As a reminder, please remember to add the following to the top of this and all other .rb files you create:

```ruby
# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
```